Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11-bullseye
```

Comment:
And latest stable of course

Reply:
Except Alpine I would say, except if really needed, as explained by tiangolo: https://github.com/tiangolo/uvicorn-gunicorn-docker#-alpine-python-warning

I assembled all this information in a specific Wiki page, because otherwise it is almost lost: https://github.com/openzim/_python-bootstrap/wiki/Dockerfile

Reply:
I meant small as in debian-slim, not alpine. There is no incentive for scrapers to be alpine-small ; but there is for offspot tools for instance